bin/yaml/julia.yaml

```
@@ -21,3 +22,7 @@ compilers:
           url: https://julialangnightlies-s3.julialang.org/bin/linux/x86_64/julia-latest-linux-x86_64.tar.gz
           strip_components: 1
           create_untar_dir: true
+    depot: {destination}/depot
+    cejl_revision: 532185e32ed03d340e16922c1da330c973047570
+    after_stage_script:
+      - JULIA_DEPOT_PATH={depot} {julia_exe} -e 'using Pkg; Pkg.add(; url="https://github.com/giordano/CompilerExplorer.jl", rev="{ce_revision}"); Pkg.precompile()'
```

```suggestion
      - JULIA_DEPOT_PATH={depot}: {julia_exe} -e 'using Pkg; Pkg.add(; url="https://github.com/giordano/CompilerExplorer.jl", rev="{ce_revision}"); Pkg.precompile()'
```

Any reason for using `{depot}` instead of `{destination}/local/share/julia`? 

We likely also need a project + manifest in the LOAD_PATH

---

I was planning adding the package to the global environment, so that we don't have to deal with adding a `Project.toml`/`Manifest.toml` here (the latter file being problematic, since it's different for each julia version).  But I'm open to better solutions.

---

So if you use the local depot https://docs.julialang.org/en/v1/base/constants/#Base.DEPOT_PATH

Then we don't need to modify the depot path later one. 

I don't think we want to use the default project since the user might try to modify it and this is read-only (might also be fine).

You need `:` after the path so that we still have the system depot available for the stdlib